examples/hello-node-express-ts/index.ts

```diff
@@ -63,6 +65,16 @@ app.get('/', async (_req: Request, res: Response, next: NextFunction) => {
     sayHello();
     res.end('Hello, World!\n');
   } catch (err) {
+    tracer.startActiveSpan('error', (span) => {
+      span.recordException(err as Error);
+      span.setStatus({
```

Comment:
I wonder if it's possible to call these two—`recordException` and `setStatus`—on an existing span generated by or around what is being tried. A separate error-only span added to the trace makes me sad.

Reply:
Whachoo think about a local const of the current span captured [right after the app.get handler function starts](https://github.com/honeycombio/honeycomb-opentelemetry-node/pull/168/files#diff-cd4e0c4ee20d5783c37d3d3563bd24d5334b59fd951986f57d4e79b43325c36cL21) and use that span here in the `catch` block?

Reply:
OK I did another update based on what we chatted about. Let me know what you think of this.

with error at the top of the try/catch block:
![trace-with-error](https://user-images.githubusercontent.com/29520003/222754442-e68bcfb8-6ab3-4d1f-8e31-864dba89202a.png)

without error:
![trace-without-error](https://user-images.githubusercontent.com/29520003/222754451-cf3d6df6-db6c-408f-89bf-5698c5555484.png)